dockerfiles/che/Dockerfile.centos

```diff
@@ -8,17 +8,16 @@
 #   Dharmit Shah  - Initial implementation
 #   Mario Loriedo - Improvements
 #
-# To build it, run in the current folder after executing build.sh:
-#  `docker build -t registry.centos.org/eclipse/che-server -f Dockerfile.centos .`
+# To build it, in current folder:
+#  cp Dockerfile.centos Dockerfile
+#  ./build.sh
 #
 # To run it:
-#  docker run --net=host \
-#             --name che \
-#             -v /var/run/docker.sock:/var/run/docker.sock \
-#             -v /home/user/che/lib:/home/user/che/lib-copy \
-#             -v /home/user/che/workspaces:/home/user/che/workspaces \
-#             -v /home/user/che/storage:/home/user/che/storage \
-#             registry.centos.org/eclipse/che-server
+#  docker run -d -p 8080:8080 \
+#            --name che \
+#            -v /var/run/docker.sock:/var/run/docker.sock \
+#            -v ~/.che/workspaces:/data \
+#            eclipse/che-server:nightly
```

Comment:
James this Dockerfile is an alternative to the che-server alpine based Dockerfile. The image should be used by individuals or organizations that prefer running Che on CentOS then alpine.

Instructions in the Dockerfiles should be for developers that know what they are doing and want hack the Dockerfile (to fix a bug in the Dockerfile for example). A user that just want to run the image should pull the pre-built images available from the Dockerhub and in registry.centos.org (and run it using the CLI).